h/accounts/models.py

```diff
@@ -136,6 +147,8 @@ def _get_password(self):
         return self._password
 
     def _set_password(self, raw_password):
+        if not PASSWORD_MIN_LENGTH <= len(raw_password) <= PASSWORD_MAX_LENGTH:
```

Comment:
We didn't enforce a maximum password length before, so let's not introduce that here.